modules/eco-checking_status_of_node_maintenance_cr_tasks.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-checking_status_of_node_maintenance_cr_tasks_{context}"]
+= Checking status of current NodeMaintenance CR tasks
+
+You can check the status of current `NodeMaintenance` CR tasks.
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+* Check the status of current node maintenance tasks by running the following command:
```

Comment:
(I would add something about what is `nm`).
Check the current status of `NodeMaintenance` (or it's acronym `nm`) CR  tasks by running the following command:

Reply:
My only concern with my comment is the use of the world "acronym" where it mostly related to capital letters abbreviation and not lower case...